infer/src/IR/Procdesc.re

```diff
@@ -519,3 +519,35 @@ let node_set_succs_exn pdesc (node: Node.t) succs exn =>
     set_succs_exn_base node' [exit_node] exn
   | _ => set_succs_exn_base node succs exn
   };
+
+let get_loop_heads pdesc => {
```

Comment:
Will this work correctly for 

(1) Nested loops
(2) Loops with `break`/`continue`/`return`
(3) Loops inside of exceptional control-flow (e.g., `try { ... } catch (Exception _} { while (e1) { ... } }`)?

I ask (1)/(2) because the usual algorithm for computing loop heads in compilers is a bit more complicated than this; you need to compute the dominators of the graph first. I'm sure this simpler algorithm works in most cases, but for more complex loops I'm guessing you'll need the classic algorithm.

I ask (2) because I think `Node.get_succs` only returns the non-exceptional successors of the current node, which might cause the algorithm to miss loop heads inside of exceptional control-flow.